app/__init__.py

```diff
@@ -2,48 +2,40 @@
 import random
 import string
 import uuid
+
+from app.celery.celery import NotifyCelery
+from app.clients import Clients
+from app.clients.document_download import DocumentDownloadClient
+from app.clients.email.aws_ses import AwsSesClient
+from app.clients.email.sendgrid_client import SendGridClient
+from app.clients.sms.aws_sns import AwsSnsClient
+from app.clients.performance_platform.performance_platform_client import PerformancePlatformClient
+from app.dbsetup import RoutingSQLAlchemy
+from app.encryption import Encryption
+
 from dotenv import load_dotenv
 
 from flask import _request_ctx_stack, request, g, jsonify, make_response
-from flask_sqlalchemy import SQLAlchemy as _SQLAlchemy
 from flask_marshmallow import Marshmallow
 from flask_migrate import Migrate
-from time import monotonic
+
 from notifications_utils.clients.zendesk.zendesk_client import ZendeskClient
 from notifications_utils.clients.statsd.statsd_client import StatsdClient
 from notifications_utils.clients.redis.redis_client import RedisClient
 from notifications_utils import logging, request_helper
+
+from time import monotonic
+
 from werkzeug.exceptions import HTTPException as WerkzeugHTTPException
 from werkzeug.local import LocalProxy
 
-from app.celery.celery import NotifyCelery
-from app.clients import Clients
-from app.clients.document_download import DocumentDownloadClient
-from app.clients.email.aws_ses import AwsSesClient
-from app.clients.email.sendgrid_client import SendGridClient
-from app.clients.sms.aws_sns import AwsSnsClient
-from app.clients.performance_platform.performance_platform_client import PerformancePlatformClient
-from app.encryption import Encryption
 
 DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
 DATE_FORMAT = "%Y-%m-%d"
 
 load_dotenv()
 
-
-class SQLAlchemy(_SQLAlchemy):
-    """We need to subclass SQLAlchemy in order to override create_engine options"""
-
-    def apply_driver_hacks(self, app, info, options):
-        super().apply_driver_hacks(app, info, options)
-        if 'connect_args' not in options:
-            options['connect_args'] = {}
-        options['connect_args']["options"] = "-c statement_timeout={}".format(
-            int(app.config['SQLALCHEMY_STATEMENT_TIMEOUT']) * 1000
-        )
-
-
-db = SQLAlchemy()
```

Comment:
This went into the `dbsetup.py` module. I am not a fan of the name I picked so I'm opened to suggestions.